app/src/main/java/com/globant/practice/presentation/view/fragment/WebClientFragment.java

```diff
@@ -27,7 +30,7 @@
     private static final String HTML_URL_KEY = "url";
     private static final String DETAIL_TYPE_KEY = "type";
     private WebView webClientWebView;
-    private ProgressDialog getWebPageIndicator;
+    private ProgressDialog webPageIndicator;
```

Comment:
This name doesn't really help, at first sight we do not know what kind of indicator is.
I'd rather name it *webPageLoadingIndicator*